### ISSUE TITLE
Migrate db failed due to null value in column "locale_id" violates not-null constraint

### ISSUE THREAD

**Issue by michael-yin** [wagtail/wagtail#6640]
### Issue Summary

When migrate db on fresh db with Wagtail 2.11, I got this error

django.db.utils.IntegrityError: null value in column "locale_id" violates not-null constraint

```
  Applying wagtailcore.0056_page_locale_fields_populate... OK
  Applying wagtailcore.0057_page_locale_fields_notnull... OK
  Applying wagtailcore.0058_page_alias_of... OK
  Applying wagtailcore.0059_apply_collection_ordering... OK
  Applying taggit.0001_initial... OK
  Applying wagtailimages.0001_squashed_0021... OK
  Applying wagtailimages.0022_uploadedimage... OK
  Applying home.0001_initial... OK
  Applying home.0002_create_homepage...Traceback (most recent call last):
  File "/home/vagrant/.virtualenvs/myblog/lib/python3.6/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.NotNullViolation: null value in column "locale_id" violates not-null constraint
DETAIL:  Failing row contains (3, 00010001, 2, 0, Home, home, t, f, /home/, , f, , null, null, f, 4, null, f, null, null, null, null, Home, null, null, e19f8aa8-40e9-4f94-8daa-3f0caed00197, null, null).
```

### Technical details

It seems this issue is related with Django migration order.

The right order should be 

```
  Applying wagtailcore.0034_page_live_revision... OK
  Applying wagtailcore.0035_page_last_published_at... OK
  Applying wagtailcore.0036_populate_page_last_published_at... OK
  Applying wagtailcore.0037_set_page_owner_editable... OK
  Applying wagtailcore.0038_make_first_published_at_editable... OK
  Applying wagtailcore.0039_collectionviewrestriction... OK
  Applying wagtailcore.0040_page_draft_title... OK
  Applying home.0001_initial... OK
  Applying home.0002_create_homepage... OK
  Applying sessions.0001_initial... OK
```

### Solution

I fixed this by adding to `home/migrations/0002_create_homepage.py` The `home` app is generated by `wagtail start` command.

```
run_before = [
    ('wagtailcore', '0041_group_collection_permissions_verbose_name_plural'),
]
```

**Comment by zerolab** [comment]
Hey @michael-yin,

Did you try this with 2.11.3? This should have been fixed by #6557 / #6559

**Comment by michael-yin** [comment]
@zerolab 

Thanks for your quick response.

This issue can be closed now.

Thx.
